integration/utils/Window.js

```diff
@@ -0,0 +1,27 @@
+class Window {
+  constructor(driver) {
+    this.driver = driver;
+  }
+
+  static async getDefaultWindow(driver, name) {
```

Comment:
Can we comment on what the different between this 2 methods? I also think it is better to have a method to create Window and another one to move to another window.

Reply:
>I also think it is better to have a method to create Window and another one to move to another window.

1. `createNewWindow(driver, name)` creates a new window. 
2. The `run(commands)` function automatically switches the window to the target one before executing the commands. So no need to have a method to switch the window.
  ```js
    async run(commands) {
      await this.driver.switchTo().window(this.handle);
      await commands();
    }
  ```